Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/README.md

```diff
@@ -2556,6 +2581,14 @@ File.Read.All
 | 2019-04-03T04:10:18.1001071Z | 2020-03-23T09:24:54.169574Z | 1355899 | 0 | lsass.exe |
 
 
+##### File Indicator Example
+
+The basic indicator fields from the command are shown below without auto extraction and enrichment.
```

Comment:
let's discuss this

Reply:
Sentence removed (as we discussed in person)